spec/models/dex_connector_oidc_spec.rb

```diff
@@ -0,0 +1,83 @@
+require "rails_helper"
+
+describe DexConnectorOidc, type: :model do
+  good_provider_url = "http://your.fqdn.here:5556/dex"
+  malformed_urls = [
+    "ftp://fqdn.is.invalid",
+    "",
+    "http:://x",
+    "fqdn.is.invalid",
+    "http://user:fqdn.is.invalid/"
+  ]
+  bad_but_well_formed_http_urls = [
+    "http://fqdn.is.invalid",
+    "http://user@fqdn.is.invalid/",
+    "https://user:pass@fqdn.is.invalid/",
+    # "https://192.0.2.0",      # RFC5737 TEST-NET-1, should trigger timeout
```

Comment:
Does this line need to be here?

Reply:
Whoops.  I was going to investigate reducing the timeout in the test cases, then uncomment that one.